L1Trigger/L1TGlobal/src/CaloCondition.cc

```diff
@@ -525,6 +525,25 @@ const bool l1t::CaloCondition::checkObjectParameter(const int iCondition,
     LogDebug("L1TGlobal") << "\t\t l1t::Candidate failed isolation requirement" << std::endl;
     return false;
   }
+
+  // sanity check LLP DISP ( bit check ) with two-state LUT = {0,1}
+  // note that this check is only valid when hwQual contains just the single LLP DISP bit, which is consistent
+  //      with the uGT firmware expectations.  If hwQual is later defined to contain more quality bits, then
+  //      the below sanity check will have to be changed to include the extra quality bits.
+  if (cand.hwQual() > 1) {
```

Comment:
Were is this `hwQual` for the `L1Candidate` set?
Wouldn't it be safer and better portable to check the explicit LLP DISP bit?